mixer/pkg/config/crd/store.go

```diff
@@ -111,13 +110,18 @@ type Store struct {
 var _ store.Backend = &Store{}
 var _ probe.SupportsProbe = &Store{}
 
+// Close implements io.Closer
+func (s *Store) Close() error {
+	close(s.donec)
+	return nil
+}
+
 // checkAndCreateCaches checks the presence of custom resource definitions through the discovery API,
 // and then create caches through lwBUilder which is in kinds. It retries within the timeout duration.
```

Comment:
Update comments? There is no timeout anymore.

Reply:
done